analyzer/src/main/resources/init.gradle

```diff
@@ -113,6 +113,17 @@ class DependencyTreePlugin implements Plugin<Gradle> {
 
             List<String> errors = []
 
+            def gradleVersion = project.gradle.gradleVersion.split('\\.')
+
+            if (gradleVersion[0].toInteger() < 3
+                    || (gradleVersion[0].toInteger() == 3 && gradleVersion[1].toInteger() < 3)) {
+                return new DependencyTreeModelImpl(project.group ?: '', project.name, project.version ?: '', [], [],
+                        [
```

Comment:
Nit: Isn't the formatting a bit weird here? I would have expected `[` / `]` to direct go before / after the inner `(` / `)`. Just wondering, not important anyway.

Reply:
Done.